libs/migrations/src/scripts/token-ids.migration.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Migration } from '..';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Token } from '@sputnik-v2/token';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class TokenIdsMigration implements Migration {
```

Comment:
There is initial token aggregation when aggregator starts, during that aggregation we load all dao tokens just in case if for some time service was turned off and there are too much transaction missed to aggregate it in regular way.